config/clients/python/template/requirements.mustache

```diff
@@ -1,5 +1,5 @@
 six >= 1.16
-setuptools >= 69.0.2
+setuptools >= 67.7.2
```

Comment:
Want to confirm this is an intended downgrade

Reply:
Yep - had to do it earlier as it was blocking the previous release: https://github.com/openfga/python-sdk/pull/46/commits/37ba09edae089a0f02c8f82c815fca5889edfb83